.github/workflows/test.yml

```
@@ -0,0 +1,21 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: "Go ${{ matrix.go }}"
```

```suggestion
    name: "Tests"
```

---

That duplicates workflow-level `Tests` name and will appear in UI as `Tests / Tests` which is a bit confusing. IMHO it is better to use command name for clarity, e.g. `Tests / go test`
```suggestion
    name: "go test"
```

---

I'm fine with `go test`